index.js

```diff
@@ -3,190 +3,226 @@
  *
  * High-level audio container
  */
-'use strict';
+'use strict'
+
+const Emitter = require('events')
```

Comment:
We have to manage creation/deletion of emitter holder then and cannot that simply inherit `EventEmitter`. `events` is not that bad I assure (I used to play around with different implementations [here](https://github.com/dfcreative/emmy) and [there](https://github.com/dfcreative/enot)), also it is native in node. Sadly it has `off` method named as `removeEventListener`.
mitt looks nice though.
I am afraid we may use it already somewhere in deps